src/firecracker/src/api_server/request/hotplug.rs

```diff
@@ -22,7 +22,7 @@ mod tests {
 
     use hotplug::parse_put_hotplug;
     use vmm::rpc_interface::VmmAction;
-    use vmm::vmm_config::hotplug::{HotplugVcpuConfig, HotplugRequestConfig};
+    use vmm::vmm_config::hotplug::{HotplugRequestConfig, HotplugVcpuConfig};
```

Comment:
I assume you fixed here the ordering of the types (there appears a change in the third commit). Please fix the errors in the commits that introduced them.